### PR TITLE
feat: output data dir with charm where

### PR DESCRIPTION
**Pull request by bashbunni**
## Changes:
- [x] add cobra command for `where` to get the data directory on the user's machine

## Related Issues:
#151 

## Other:
Example output:
output

### CODE REVIEW

File: cmd/where.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	Use: "where",
+	Short: "Find where your cloud.charm.sh folder resides on your machine",
+	Long: paragraph("Find the absolute path to your charm keys, databases, etc."),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cc := initCharmClient()
+		path, err := cc.DataPath()
+		if err != nil {
+			return err
+		}
+		fmt.Fprint(cmd.OutOrStdout(), path)
```
Comment (toby) [inline cmd/where.go:19]: Looks good! I'd put a new line at the end of this though, so the prompt is on the next line after running `charm where`.

Comment (bashbunni) [reply cmd/where.go:19]: Can do!

**Comment by bashbunni** [comment]
Updated to include new line:
output

**Comment by toby** [comment]
looks good!

**Review by toby — APPROVED** [review]

**Review by muesli — APPROVED** [review]

**Pull request merged**